ccg_nlpy/pipeline_base.py

```diff
@@ -51,3 +51,7 @@ def call_server(text, views):
         logger.error("This function should be overrided.")
         #raise NotImplementedError()
         return None
+
+
+    def clean_text(self, text: str) -> str:
```

Comment:
This is the syntax in Python 3, that is why semaphore failed (it is using Python 2). And it seems like Python 3 won't enforce the parameters to have the same type as specified https://stackoverflow.com/questions/2489669/function-parameter-types-in-python .